app/main/views/communications.py

```diff
@@ -53,15 +42,19 @@ def upload_communication(framework_slug):
             errors['communication'] = 'not_open_document_format_or_csv'
 
         if 'communication' not in errors.keys():
-            _save_file(communications_bucket, framework_slug, 'updates/communications', the_file, 'communication')
+            path = "{}/communications/updates/communications/{}".format(framework_slug, the_file.filename)
+            communications_bucket.save(path, the_file, acl='private', download_filename=the_file.filename)
+            flash('communication', 'upload_communication')
 
     if request.files.get('clarification'):
         the_file = request.files['clarification']
         if not file_is_pdf(the_file):
             errors['clarification'] = 'not_pdf'
 
         if 'clarification' not in errors.keys():
-            _save_file(communications_bucket, framework_slug, 'updates/clarifications', the_file, 'clarification')
+            path = "{}/communications/updates/clarifications/{}".format(framework_slug, the_file.filename)
+            communications_bucket.save(path, the_file, download_filename=the_file.filename)
```

Comment:
I think both should have private ACL maybe? I don't see any reason why we'd have one public and one private.

Reply:
Cool, just looked at the ticket and `agreements` is implied too. Will update when the tests are merged https://github.com/alphagov/digitalmarketplace-admin-frontend/pull/302

Reply:
I meant both communications and clarifications. Agreements are already private (see https://github.com/alphagov/digitalmarketplace-admin-frontend/blob/master/app/main/views/suppliers.py#L277)

Reply:
Derp, I also this